plugins/common/tls/config.go

```diff
@@ -15,7 +15,8 @@ type ClientConfig struct {
 	TLSCA              string `toml:"tls_ca"`
 	TLSCert            string `toml:"tls_cert"`
 	TLSKey             string `toml:"tls_key"`
-	TLSKeyPwd          string `toml:"tls_key_pwd"`
+	TLSCertAndKey      string `toml:"tls_cert_key"`
+	TLSKeyPassword     string `toml:"tls_key_password"`
```

Comment:
Warning, this introduces a breaking change! Users who configured `tls_key_pwd` before will have errors when updating telegraf. Please make use of the [Deprecate options](https://github.com/influxdata/telegraf/blob/master/docs/developers/DEPRECATION.md#deprecate-options) functionalities.